### ISSUE TITLE
@auth/sveltekit@0.3.4 fails to run.  Can't find package $app

### ISSUE THREAD

**Issue by rory-orennia** [nextauthjs/next-auth#8004]
### Environment

System:
    OS: macOS 13.4
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 494.84 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 19.7.0 - ~/.nvm/versions/node/v19.7.0/bin/node
    Yarn: 1.22.19 - ~/.yarn/bin/yarn
    npm: 9.5.0 - ~/.nvm/versions/node/v19.7.0/bin/npm
    pnpm: 8.6.6 - ~/Library/pnpm/pnpm
  Browsers:
    Brave Browser: 114.1.52.129
    Edge: 114.0.1823.55
    Safari: 16.5

### Reproduction URL

https://authjs.dev/reference/sveltekit

### Describe the issue

Following the examples at https://authjs.dev/reference/sveltekit, the app will return 500.  Reverting to 0.3.3 fixes the issue.  Must be related to removing the dependency on sveltejs

```
11:50:39 a.m. [vite] Error when evaluating SSR module /src/hooks.server.ts: failed to import "@auth/sveltekit"
|- Error [ERR_MODULE_NOT_FOUND]: Cannot find package '$app' imported from /Users/myuser/git/someProject/node_modules/.pnpm/@auth+sveltekit@0.3.4_@sveltejs+kit@1.22.1/node_modules/@auth/sveltekit/index.js
```

### How to reproduce

1. follow the steps on https://authjs.dev/reference/sveltekit
2. run the dev server
3. open the page and get a 500

### Expected behavior

Auth check should work and page should load

**Comment by torablien** [comment]
+1, was just filing this issue myself. reproducible on real project and dummy example. Easy repro steps:

1. Create skeleton Sveltekit app: `npm create svelte@latest myApp`
2. Install authjs for sveltekit: `npm install @auth/core @auth/sveltekit`
3. Import Sveltekit Auth in `hooks.server.ts`: `import { SvelteKitAuth } from "@auth/sveltekit";`
4. Run and visit localhost: `npm run dev`


Seemingly a result of #7989.

**Comment by odysseaspapadimas** [comment]
Just run into this issue as well, quite a surprise to see an issue from 6 minutes ago, any solutions?

**Comment by torablien** [comment]
> odysseaspapadimas

Downgrade to 0.3.3 fixes it.

**Comment by balazsorban44** [comment]
Fixed in 0.3.5, please upgrade! 🙏

**Comment by rory-orennia** [comment]
Works for me now! Thanks for the quick fix!
